### ISSUE TITLE
accessibilityLabel for Picker not read by VoiceOver

### ISSUE THREAD

**Issue by mitchdowney** [react-native-picker/picker#310]
@react-native-picker/picker 1.16.5
iOS 14

I can't get the accessibilityLabel to be read by VoiceOver on an iOS device with a simple Picker example. Instead it is reading the currently selected PickerItem for the focused Picker. Am I doing something wrong? Or does the accessibilityLabel prop on the Picker not get read by VoiceOver?

In the example below I would like the word "hours" read before

In the render:
```
<Picker
  accessibilityLabel={translate('hours')}
  enabled={!isActive}
  onValueChange={(itemValue) => {
    handleUpdateSleepTimer(itemValue, currentMinute, currentSecond)
  }}
  selectedValue={currentHour}>
  {hourItems}
</Picker>
```

How the `hourItems` is defined:
```
const generatePickerNumberItems = (total: number, key: string) => {
  const arr = [] as any
  for (let i = 0; i < total; i++) {
    arr.push(<Picker.Item key={i + key} label={i.toString()} value={i} />)
  }

  return arr
}

const hourItems = generatePickerNumberItems(24, 'hourItems')
```

It also seems that I cannot change the accessibilityLabel or accessibilityHint for the PickerItems, so I have no way of announcing to the user this Picker value represents the selected number of hours.

![IMG_5013](https://user-images.githubusercontent.com/2262919/130314837-dacec9a6-e4b7-4ec1-a3a1-10ebbdb7c230.png)

**Comment by mitchdowney** [comment]
I just updated our app to use @react-native-picker/picker 2.4.0, and I'm still running into this issue. It's making the Sleep Timer in Podverse almost unusable for screen reader users, as we can't announce each individual "hours" "minutes" "seconds" column.

Are we the only ones running into this issue? Other than setting an accessibilityLabel to the <Picker /> component, I'm not sure what else we're supposed to do to make this work.

**Comment by duraz0rz** [comment]
It looks like accessibility-related props aren't being added to the final picker element at all. This is what I get as the rendered JSX when I was trying to write tests using `accessibilityLabel`.

Test:
```
    it("loads a list of songs from the server", async () => {
      const { debug, getByHintText } = await subject()

      debug("subject output")

      const dropdown = getByHintText("Pick a song")

      expect(dropdown.children).toHaveLength(songs.length)
    })
```

Code under test:
```
  ... //snip
  return (
    <>
      <Text>Pick a song...</Text>
      <Picker testID="song-picker" accessibilityLabel="Pick a song">
        {listOfSongs}
      </Picker>
    </>
  )
```

Output of `debug()` in the test:
```
     Array [
      <Text>
        Pick a song...
      </Text>,
      <View>
        <RNCPicker
          items={
            Array [
              Object {
                "label": "hello",
                "testID": "hello",
                "textColor": undefined,
                "value": "hello",
              },
              Object {
                "label": "Goodbye",
                "testID": "Goodbye",
                "textColor": undefined,
                "value": "Goodbye",
              },
            ]
          }
          numberOfLines={1}
          onChange={[Function anonymous]}
          selectedIndex={0}
          style={
            Array [
              Object {
                "height": 216,
              },
              undefined,
            ]
          }
          testID="song-picker"
        />
      </View>,
    ]
```

**Comment by paperbackdragon** [comment]
I'm also running into this issue, using the Picker with React Native Windows. Any `accessibilityLabel` I pass it is not being read by Narrator.

**Comment by paperbackdragon** [comment]
@Naturalclar I believe this still might be an issue for iOS, since I only really fixed it for Windows. Would you be able to re-open it?

**Comment by ha3** [comment]
Hi @Naturalclar I can confirm that this issue still persists with 2.4.4

**Comment by Naturalclar** [comment]
Reopening it for iOS

**Comment by kastwey** [comment]
@Naturalclar Yeaaaah! ¡Tested with Voice over and it works like a charm! Thanks!!!!!!!!!!!!!
